Release Notes/2018/2018-09-04-cloud-platform-release-notes.md

```diff
@@ -0,0 +1,42 @@
+{{{
+"title": "Cloud Platform - Release Notes: September 04, 2018",
+"date": "09-04-2018",
+"author": "Matt Schwabenbauer",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+### Enhancements (6)
+
+#### [Cloud Application Manager](//www.ctl.io/cloud-application-manager/)
+
+#### [Application Lifecycle Management](//www.ctl.io/cloud-application-manager/application-lifecycle-management/)
+
+##### Support for Azure Scale Sets
+
+Application Lifecycle Management now provides support to use Scale Set into a Microsoft Azure Deployment Policy box, so that the instances deployed with it would be included into a Scale Set Azure resource and enable the user to turn on the auto scaling feature by setting up the CPU thresholds and instance increments.
+
+##### Support for Azure Load Balancers
+
+Application Lifecycle Management now supports Azure Load Balancer and Azure App Gateway to be added to a Microsoft Azure Deployment Policy box. Once a user enables Scale Set on it, a new section appears allowing the user to enable load balancer, either by using Azure Load Balancer or by using an Azure App Gateway.
+
+#### [SafeHaven DRaaS](//www.ctl.io/disaster-recovery/)
+
+* Support for Linux Operating Systems : RHEL6/7, Ubuntu 14/16 for DR to Azure.
+* CMS box in CAM catalogs to deploy and configure SafeHaven Console.
```

Comment:
@MattSchwabby Can you please make a minor change? Please expand CAM to "Cloud Application Manager"